feature-libs/storefinder/components/store-finder-stores-count/store-finder-stores-count.component.ts

```diff
@@ -23,4 +28,9 @@ export class StoreFinderStoresCountComponent implements OnInit {
     this.locations$ = this.storeFinderService.getViewAllStoresEntities();
     this.isLoading$ = this.storeFinderService.getViewAllStoresLoading();
   }
+
+  navigateToLocation(isoCode: string, event?: Event): void {
+    event?.preventDefault();
```

Comment:
Because of `preventDefault()` potentially blocking default `routerLink` behaviour, we surround this functionality with the condition of `this.routingService` being truthy to avoid regression.